scanner/e2etests/testdata/image_tests.json

```diff
@@ -5067,14 +5068,14 @@
                 "Name": "python3-urllib3",
                 "NamespaceName": "rhel:9",
                 "VersionFormat": "rpm",
-                "Version": "1.26.5-3.el9.noarch",
+                "Version": "1.26.5-3.el9",
                 "AddedBy": "sha256:2412e60e610160d090f7e974a208c6ffd26b2d530361b7c9aa8967e160ac7996"
             },
             {
                 "Name": "vim-minimal",
                 "NamespaceName": "rhel:9",
                 "VersionFormat": "rpm",
-                "Version": "2:8.2.2637-16.el9_0.2.x86_64",
```

Comment:
Why are we removing the architecture here?

Reply:
Because ClairCore doesn't return RPM's EVRA (name-version-release-arch), just the EVR (name-version-release):

https://github.com/quay/claircore/blob/b15bdcb774c795a7f4a0ac4f57127eee1613dd24/rpm/native_db.go#L177-L186

Reply:
can you make a Jira to address this? Doesn't need to be done for GA, but we added the arch for a reason: I remember finding two components in the same image that looked almost identical but they were built for different arches, and it was hard to differentiate them. Perhaps that's not an issue anymore or it's not important enough, but worth looking into

Reply:
FYI, ClairCore has the arch info in the index content. The question is if we want to modify our index content to have that arch in the version or leave Central to put it where it is meaningful. Central seems more reasonable.

What I will do is I will modify the tests to match arch. This can be done in this PR.